.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
@gwaardenburg Geoff, do you know which extension adds that? I don't get the same thing in my vscode.

Reply:
I do not, I actually missed that this had gone in the MR, I can remove it if you like? I think my VSCode updated itself this morning so it might be related to that